src/gf-platformid

```diff
@@ -52,7 +52,9 @@ coreos_gf upload "${tmpd}"/bls.conf "${blscfg_path}"
 if [ "$basearch" = "s390x" ] ; then
     # Before we re-run zipl make sure we have the firstboot options
     # A hack similar to https://github.com/coreos/coreos-assembler/blob/master/src/create_disk.sh#L381
-    sed -i -e 's,^\(options .*\),\1 ignition.firstboot,' "${tmpd}"/bls.conf
+    coreos_gf download /boot/ignition.firstboot "${tmpd}"/ignition.firstboot
+    IGN_NET_KARGS=$(sed 's/^set ignition_network_kcmdline=.\(.*\).$/\1/' "${tmpd}"/ignition.firstboot)
+    sed -i -e 's,^\(options .*\),\1 ignition.firstboot '"${IGN_NET_KARGS}"',' "${tmpd}"/bls.conf
```

Comment:
need to use another delimiter and not `,` as the cmdline has `ip=dhcp,dhcp6` and throws:
```
sed: -e expression #1, char 62: unknown option to `s'
```
I used `|` and it worked:

```
sed -i -e 's|^\(options .*\)|\1 ignition.firstboot rd.neednet=1 ip=dhcp,dhcp6|' test.conf
```